data/nasty.csv

```diff
@@ -0,0 +1,10 @@
+﻿LABEL,X,ZERO,MISS,BIG,ONE,HUGE,TINY,ROUND
```

Comment:
this data is small - I think it will be easier to just embed it in a test set to keep things in one place.

Reply:
Thanks for your suggestion. Updated the code accordingly